server/controller/routes/portal/portalLogainPage.js

```diff
@@ -29,7 +28,7 @@ const getHashedPinCode = (req, res, next) => {
 const createPortalToken = (req, res, next) => {
   const token = jwt.sign({ id: req.adminId }, process.env.SECRET_KEY_PORTAL);
   res.cookie('portalToken', token);
-  res.json({ msg: 'Portal, logged in successfully' });
+  res.json({ msg: 'Portal, logged in successfully', status: 301 });
```

Comment:
if you stick to what I mentioned above then you won't need the 301 status 

```suggestion
  res.json({ msg: 'logged in successfully' });
```